compiler/quilt/tools/command.py

```diff
@@ -82,16 +82,23 @@ class CommandException(Exception):
     """
     pass
 
-def parse_package_extended(name):
+
+#return type for parse_package_extended
+PackageInfo = namedtuple("PackageInfo", "full_name, team, user, name, subpath, hash, version, tag")
+def parse_package_extended(identifier):
     """
     Parses the extended package syntax and returns a tuple of (package, hash, version, tag).
     """
-    match = EXTENDED_PACKAGE_RE.match(name)
+    match = EXTENDED_PACKAGE_RE.match(identifier)
     if match is None:
         pkg_format = '[team:]owner/package_name/path[:v:<version> or :t:<tag> or :h:<hash>]'
         raise CommandException("Specify package as %s." % pkg_format)
 
-    return match.groups()
+    full_name, hash, version, tag = match.groups()
```

Comment:
Let's use `pkg_hash` instead of `hash`, just to avoid overriding the `hash` builtin.